packages/core/src/util/config.ts

```diff
@@ -238,6 +238,10 @@ export class Config {
     return getEnvVarOrFail("WRITE_TO_S3_QUEUE_URL");
   }
 
+  static isSftpActionLambda(): boolean {
+    return getEnvVar("SFTP_ACTION_LAMBDA") != undefined;
+  }
+
```

Comment:
My understanding that we want to keep this file as much the following as possible.
```
  static getWriteToS3QueueUrl(): string {
    return getEnvVarOrFail("WRITE_TO_S3_QUEUE_URL");
  }
```

I had a debate with raf over this, so I'd create this:
```
  static getSftpActionLambda(): boolean {
    return getEnvVar("SFTP_ACTION_LAMBDA") != undefined;
  }
```

And then call it with the logic in some sftp lambda specific directory.